mks_server.py

```diff
@@ -184,17 +190,23 @@ def update_readings(self):
         for G in self.gauges:
             try:
                 result = yield G['packet'].send()
-            except Exception:
+            except Exception as e:
+                logging.debug('Serial return error: {}'.format(e))
```

Comment:
to log exceptions, you can just add an `exc_info` keyword arg (see [here](https://docs.python.org/2/library/logging.html#logging.Logger.debug)):
```python
except Exception:
    logging.debug('Serial return error', exc_info=True)
```